DataConnectors/Citrix_ADC_WAF.json

```diff
@@ -0,0 +1,116 @@
+{
+    "id": "Citrix",
+    "title": "Citrix ADC WAF",
+    "publisher": "Citrix System",
+    "descriptionMarkdown": "Common Event Format (CEF) is an industry standard format on top of Syslog messages, used by many security vendors to allow event interoperability among different platforms. By connecting your CEF logs to Azure Sentinel, you can take advantage of search & correlation, alerting, and threat intelligence enrichment for each log.",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "CitrixWafLogs",
```

Comment:
Would be great to update the legend to be "Citrix Netscaler WAF" so that there's context.

Reply:
o be consistent with the title in azure marketplace using CitrixWafLogs here